cmd/lima-guestagent/install_systemd_linux.go

```diff
@@ -49,6 +49,7 @@ func installSystemdAction(cmd *cobra.Command, _ []string) error {
 	args := [][]string{
 		{"daemon-reload"},
 		{"enable", "--now", "lima-guestagent.service"},
+		{"try-restart", "lima-guestagent.service"},
```

Comment:
This was needed to properly upgrade on first start.

Without this, the below case was failing
lima 0.15 create and start -> stop -> lima 1 start -> guest is updated but not restarted to pick up latest one -> This will be taken only after second restart